catroid/res/values/strings.xml

```diff
@@ -28,6 +28,8 @@
     <string name="cancel_button">Cancel</string>
     <string name="rename">Rename</string>
     <string name="delete">Delete</string>
+    <string name="move_down">Move Down</string>
+    <string name="move_up">Move Up</string>
```

Comment:
Maybe rename the **string ids** to something more clarifying, e.g. `menu_item_move_up` + `down`
This also helps translators to understand the context of the strings better.